specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/examples/Sessions_UpdateEnvironmentVariables.json

```diff
@@ -0,0 +1,48 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "sessionPoolName": "testsessionpool",
+    "name": "testsession",
+    "api-version": "2024-02-02-preview",
+    "environmentVariables": {
+      "environmentVariables": [
+        {
+          "key": "key1",
+          "value": [
+            {
+              "name": "name-1-1",
+              "value": "value-1-1",
+              "secretRef": "secretRef-1-1"
```

Comment:
`value` and `secretRef` should be mutual exclusive I suppose?

Reply:
Thanks, fixed